compute/cloud-client/src/test/java/compute/windows/osimage/WindowsOsImageIT.java

```diff
@@ -55,94 +54,91 @@ public class WindowsOsImageIT {
   private static final String ZONE = getZone();
   private static final int MAX_ATTEMPT_COUNT = 3;
   private static final int INITIAL_BACKOFF_MILLIS = 120000; // 2 minutes
-  private static String INSTANCE_NAME;
-  private static String DISK_NAME;
-  private static String IMAGE_NAME;
+  private static String testInstanceName;
+  private static String testImageName;
+
+  private static String getBootDiskName(String instanceName) {
+    return instanceName + "-boot-disk";
+  }
+
   @Rule
   public final MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(
       MAX_ATTEMPT_COUNT,
       INITIAL_BACKOFF_MILLIS);
-  private ByteArrayOutputStream stdOut;
-
-  // Check if the required environment variables are set.
-  public static void requireEnvVar(String envVarName) {
-    assertWithMessage(String.format("Missing environment variable '%s' ", envVarName))
-        .that(System.getenv(envVarName)).isNotEmpty();
-  }
-
-  @BeforeAll
-  public static void setup()
-      throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    final PrintStream out = System.out;
-    ByteArrayOutputStream stdOut = new ByteArrayOutputStream();
-    System.setOut(new PrintStream(stdOut));
 
-    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
-    requireEnvVar("GOOGLE_CLOUD_PROJECT");
-
-    // Cleanup existing test instances.
-    Util.cleanUpExistingInstances("windowsimage-test-instance-", PROJECT_ID, ZONE);
```

Comment:
Should this line be retained? It will ensure that instances from any previous failed runs are cleaned up.

Reply:
This line is removed following the best practice of "fail sooner". The tests should clean up the environment. If they don't we want means to discover it. This clean up line, ran in the test `setup()` does not let us to do it.